packaging/rpm/microshift.spec

```
@@ -117,6 +117,7 @@ Things break. But you can still help shape it, too.
 Summary: SELinux policies for MicroShift
 BuildRequires: selinux-policy
 BuildRequires: selinux-policy-devel
+BuildRequires: container-selinux
```

```suggestion
Requires: container-selinux
```

If we require it at runtime (and not build)